code/modules/atmospherics/machinery/components/fusion/hfr_main_processes.dm

```diff
@@ -287,6 +287,7 @@
 	switch(power_level)
 		if(1)
 			if(moderator_list[/datum/gas/plasma] > 100)
+				internal_output.assert_gases(/datum/gas/nitrous_oxide)
 				moderator_internal.gases[/datum/gas/nitrous_oxide] += scaled_production * 0.5
```

Comment:
this code is just incorrect, it's adding to nothing. This will produce undefined behavior

Reply:
also why do we assert on internal output for this if we don't access it at all? is this for later?

Reply:
I was careless, looks like sometimes the gas is added straight to the moderator, sometimes it's added to internal_output. I pushed a commit claiming a "fix", but I don't know if it's intended behavior to dump some gases straight into the moderator. Let me know if straight-to-moderator dumping should be kept.